test/unit/collaborators.test.js

```diff
@@ -1,16 +1,173 @@
+  readmeNoCollaborators,
+  readmeNoCollaboratorE,
+  readmeUnordered,
+  collaborators
+} = require('../fixtures/data');
 const TestLogger = require('../fixtures/test_logger');
-const logger = new TestLogger();
-const assert = require('assert');
+const assertThrowsAsync = require('../fixtures/assert_throws_async');
 
 describe('collaborators', function() {
-  it('getCollaborators', async function() {
-    const parsed = await getCollaborators(readme, logger, 'nodejs', 'node');
-    assert.deepStrictEqual(parsed, collaborators);
+  const collaborator = collaborators.get('bar');
+  let logger = null;
+
+  beforeEach(() => {
+    logger = new TestLogger();
+  });
+
+  afterEach(() => {
+    logger.clear();
+  });
+
+  describe('Collaborator', () => {
+    describe('isActor', () => {
+      it('should return false if actor is a ghost', () => {
+        assert.strictEqual(collaborator.isActor(null), false);
+      });
+
+      it('should return false if the actor has no login', () => {
+        const collaboratorNoLogin = {
+          name: 'Bar User',
+          email: 'bar@example.com',
+          type: 'TSC'
+        };
+        assert.strictEqual(collaborator.isActor(collaboratorNoLogin), false);
+      });
+
+      it(
+        'should return true if the the collaborator and actor login are equal',
+        () => {
+          assert.strictEqual(collaborator.isActor(collaborator), true);
+        });
+    });
+
+    describe('isTSC', () => {
+      it('should return true if collaborator type is TSC', () => {
+        // Collaborator 'bar' is TSC
+        assert.strictEqual(collaborator.isTSC(), true);
+      });
+
+      it('should return false if the collaborator type is not TSC', () => {
+        // Collaborator 'foo' is not TSC
+        const collaboratorNoTSC = collaborators.get('foo');
+        assert.strictEqual(collaboratorNoTSC.isTSC(), false);
+      });
+    });
+
+    describe('getName', () => {
+      it('should return the name and login', () => {
+        collaborators.forEach(collaborator => {
+          assert.strictEqual(
+            collaborator.getName(),
+            `${collaborator.name}(${collaborator.login})`);
+        });
+      });
+    });
+
+    describe('getContact', () => {
+      it('should return the name and the email', () => {
+        collaborators.forEach(collaborator => {
+          assert.strictEqual(
+            collaborator.getContact(),
+            `${collaborator.name} <${collaborator.email}>`);
+        });
+      });
+    });
+  });
+
+  describe('getCollaborators', () => {
+    it('should return all collaborators', async function() {
+      const parsed = await getCollaborators(readme, logger, 'nodejs', 'node');
+      assert.deepStrictEqual(parsed, collaborators);
+    });
+
+    it(
+      'should throw error if there is no TSC section in the README',
+      async() => {
+        await assertThrowsAsync(
+          async() => getCollaborators(readmeNoTsc, logger, 'nodejs', 'node'),
```

Comment:
A minor nit: can you add space between `async` and `()`

Reply:
That wouldn't pass the linter, the current version is correct.